content/first-order-logic/models-theories/theories.tex

```diff
@@ -126,8 +126,8 @@
         \Part{y}{z}) \lif \Part{x}{z})]]],\\
 \intertext{Moreover, any two objects have a fusion (an object that has
   only these two objects and all their parts as parts).}  &
-\lforall[x][\lforall[y][\lexists[z][\lforall[u][(\Part{u}{z} \liff
-        (\Part{u}{x} \land \Part{u}{y}))]]]]
+\lforall[x][\lforall[y][\lexists[z][\lforall[u][(\Part{z}{u} \liff
```

Comment:
Ah you're right.  It's "Sum_1" in the SEP article.  So we should change it but also change the explanation before it, which is just wrong (the fusion also has fusions of proper parts of x and y as its parts!).  It should read something like "Moreover, any two objects  $x$ and $y$ have a sum (an object which consists of $x$ and $y$ ``fused together'' and nothing else, i.e., whatever the sum is a part of, also has $x$ and $y$ as parts, and vice versa)."